src/app/messages/MessageProvider.tsx

```diff
@@ -7,13 +7,13 @@ import createIntl from './createIntl';
 import { currentLocale } from './selectors';
 
 // tslint:disable-next-line:variable-name
-const MessageProvider = (props: { children?: React.ReactNode }) => {
+const MessageProvider = (props: React.PropsWithChildren<{}>) => {
   const services = useServices();
   const [intl, setIntl] = useState<IntlShape | null>(services.intl.current);
   const bookLocale = useSelector(currentLocale);
 
   useEffect(() => {
-    if (!bookLocale) {
+    if (!bookLocale || intl) {
```

Comment:
Would this be a problem when switching to a book with a different locale?

Reply:
Yea, that's not a thing that should ever happen but I can add some logic to test for it